lib/bumper.js

```diff
@@ -64,10 +69,27 @@ class Bumper {
         return this._createTag()
       })
       .then(() => {
+        return this._blackduck(options)
+      })
+      .then(() => {
         return this.ci.push(this.vcs)
       })
   }
 
+  _blackduck (options) {
```

Comment:
I'd like to see tests around the internals of this function. i.e. a test where the blackduck options are present and one where they are not.